src/main/java/io/antmedia/datastore/db/types/Broadcast.java

```diff
@@ -354,6 +354,9 @@ public Broadcast() {
 	 */
 	private long updateTime = 0;
 
+	@ApiModelProperty(value = "if true, only subscribers registered to this broadcast are allowed to watch.")
+	private boolean subscriberOnly = false;
```

Comment:
I think we can have this solution without using this field

Reply:
what if there is no subscriberOnly field and someone tries to watch with random subId by modifying client code

Reply:
This problem should be fixed in stream security level. Not here. Adding this field make things more complex. Just Keep it simple

Reply:
done